package.json

```diff
@@ -115,7 +117,7 @@
     "test:unit:watch": "jest --watchAll",
     "wait:test": "wait-on tcp:8080 && yarn svelte:build && yarn cypress:run",
     "wait:debug": "wait-on tcp:8080 && yarn cypress:open",
-    "dist": "rm -rf ./dist && mkdir ./dist && yarn proxy:clean && yarn svelte:clean && yarn svelte:build && yarn proxy:build:release && cp proxy/target/release/api dist/proxy && cp proxy/target/release/git-remote-rad dist && electron-builder --publish never",
+    "dist": "rm -rf ./dist && mkdir ./dist && yarn proxy:clean && yarn svelte:clean && RADICLE_UPSTREAM_EXPERIMENTAL=false NODE_ENV=production yarn svelte:build && yarn proxy:build:release && cp proxy/target/release/api dist/proxy && cp proxy/target/release/git-remote-rad dist && electron-builder --publish never",
```

Comment:
This command is used to build a binary package, so these env variables don't make much sense here.
A user will later take the generated package and install it on their system, after that they would run it from the GUI (by double clicking), without passing any env variables by default.

When running the app from a package electron automatically sets the `NODE_ENV` to `production`. This is how `isDev` works.

Reply:
Right. I have worked in other projects that were bundled and took the relevant environment variables into account, I thought the same would happen here.

 > When running the app from a package electron automatically sets the NODE_ENV to production. This is how isDev works.

I see, so `NODE_ENV` is a `electron`-reserved variable. I thought it was ours and was going to suggest namespace it better.

Reply:
Fixed, thanks.